### PR TITLE
DS-203 Add className wrapper to Spinner API

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: src/components/spinner/Spinner.tsx [hunk 1]
```diff
@@ -20,26 +20,34 @@
...
 import { keyframes } from '@emotion/react';
 import styled from '@emotion/styled';
+import classNames from 'classnames';
 import { ReactNode } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { screenReaderOnly } from '~common/helpers/styles';
 
 interface Props {
   className?: string;
+  classNameWrapper?: string;
```
Comment (david-cho-lerat-sonarsource) [inline src/components/spinner/Spinner.tsx:30]: using `wrapperClassName` would be less confusing and more accurate IMO

**Review by david-cho-lerat-sonarsource — CHANGES_REQUESTED** [review]
This isn't really adding a "large spinner", but rather a "stylable spinner", right? You can use that custom style to make it large, or anything else

**Comment by gregaubert** [comment]
> This isn't really adding a "large spinner", but rather a "stylable spinner", right? You can use that custom style to make it large, or anything else

Yeah indeed, at first I added a isLarge prop and then I noticed there were other use cases and changed the implem and forgot to rename the PR.

File: src/components/spinner/Spinner.tsx (hunk 1 above)
Comment (gregaubert) [reply src/components/spinner/Spinner.tsx:30]: Ok I'll change it!

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
LGTM, thanks! :+1:

**Pull request merged**